packages/lesmis/src/api/mutations/useUpdateFavouriteDashboardItem.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+import { useMutation } from 'react-query';
+import { debounce } from '@tupaia/utils';
+
+import { IS_NOT_FAVOURITE, IS_FAVOURITE } from '../../constants';
+import { post } from '../api';
+
+const useLikeDashboardItem = () =>
```

Comment:
```suggestion
const useFavouriteDashboardItem = () =>
```
Let's keep the terminology consistent